packages/sdk-core/src/modules/dapp-staking/index.ts

```diff
@@ -261,11 +277,11 @@ const getTxsForClaimDapp = async ({ dappAddress,
 
     if (era === e) {
       const tx = api.tx.dappsStaking.claimDapp(getDappAddressEnum(dappAddress), era);
-      transactions.push({ payload: tx, weight: info.weight });
+      transactions.push(new PayloadWithWeight(tx, info.weight));
     } else {
       // Memo: e -> skip to the era that have been staked after unstaked
       const tx = api.tx.dappsStaking.claimDapp(getDappAddressEnum(dappAddress), e);
-      transactions.push({ payload: tx, weight: info.weight });
+      transactions.push(new PayloadWithWeight(tx, info.weight));
```

Comment:
I believe there is a type error in `info.weight` after updating Polkadot.js packages. You can refer to [this](https://github.com/AstarNetwork/astar-apps/pull/687) PR after updating the packages.

Reply:
changing as per PR is giving Codec error